crates/uv/tests/it/lock.rs

```diff
@@ -10621,10 +10621,10 @@ fn lock_mixed_extras() -> Result<()> {
         async = ["iniconfig>=2"]
 
         [build-system]
-        requires = ["setuptools>=42"]
-        build-backend = "setuptools.build_meta"
+        requires = ["hatchling"]
+        build-backend = "hatchling.build"
```

Comment:
Ugh... `setuptools` puts `{name}.egg-info` inside `src`, so it bumps the cache key here (since creating a directory within a directory bumps the parent's `mtime` and `ctime`).

I could just record directory _existence_ and have the same effect? There doesn't seem to be reliable way to get directory creation time (ignoring child contents) cross-platform.

Reply:
Actually, I can probably use the inode number instead of `mtime`.